DiagnosticsExtension/Models/ConnectionStringValidator/Constants.cs

```diff
@@ -14,38 +14,51 @@ namespace DiagnosticsExtension.Models.ConnectionStringValidator
 {
     public static class Constants
     {
-        #region string constants for network validator
         public const string User = "User";
         public const string System = "System";
         public const string UnderscoreSeperator = "__";
         public const string ColonSeparator = ":";
         public const string ClientId = "__clientId";
         public const string Credential = "__credential";
-        public const string ServiceUri = "__serviceUri";
-        public const string BlobServiceUri = "__blobServiceUri";
-        public const string BlobServiceUriMissing = "Neither {0}__blobServiceUri nor {0}__serviceUri app settings were found. <a href= 'https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-trigger?tabs=in-process%2Cextensionv5&pivots=programming-language-csharp#identity-based-connections' target='_blank'>Click here to know more</a>";
-        public const string BlobServiceUriEmpty = "{0} app setting is empty. <a href= 'https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-trigger?tabs=in-process%2Cextensionv5&pivots=programming-language-csharp#identity-based-connections' target='_blank'>Click here to know more</a>";
-        public const string ServiceUriEmpty = "ServiceUriEmpty";
-        public const string QueueServiceUriMissing = "The {0}__queueServiceUri was not found or is set to a blank value. <a href= 'https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-trigger?tabs=in-process%2Cextensionv5&pivots=programming-language-csharp#identity-based-connections' target='_blank'>Click here to know more</a>";
         public const string QueueServiceUri = "__queueServiceUri";
         public const string ValidCredentialValue = "managedidentity";
         public const string FullyQualifiedNamespace = "__fullyQualifiedNamespace";
-        public const string ManagedIdentityClientIdNullorEmpty = "The {0}__clientId was not found or is set to a blank value.<a href = 'https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference?tabs=blob#common-properties-for-identity-based-connections' target = '_blank' > Click here to know more</a>";
+        public const string UnknownErrorSummary = "Validation of connection string failed due to an unknown error.";
+        public const string GenericDetailsMessage = "Additional details of the error:";
+        public const string ManagedIdentityTutorial = "Here is a <a href='https://docs.microsoft.com/azure/azure-functions/functions-identity-based-connections-tutorial' target='_blank'>relevant tutorial</a>.";
```

Comment:
Trying to keep the message short as it will some times be shown in the summary view instead of the details view.

Changed to "Refer to this relevant tutorial"